configure

```diff
@@ -23,13 +23,8 @@ function bazel_clean_and_fetch() {
   # TODO(pcloudy): Re-enable it after bazel clean --expunge is fixed.
   if ! is_windows; then
     bazel clean --expunge
-    # TODO(https://github.com/bazelbuild/bazel/issues/2220) Remove the nested `bazel query`.
-    bazel fetch $(bazel query "//tensorflow/... -//tensorflow/examples/android/...")
-  else
-    # TODO(pcloudy): Also filter out //tensorflow/examples/android/... on Windows after
-    # https://github.com/bazelbuild/bazel/issues/2248 is fixed.
-    bazel fetch //tensorflow/...
   fi
+  bazel fetch "//tensorflow/... -//tensorflow/examples/android/..."
```

Comment:
Since `bazel fetch $(bazel query "//tensorflow/... -//tensorflow/examples/android/..."` doesn't work on Windows due to https://github.com/bazelbuild/bazel/issues/2248, I replaced it with `bazel fetch "//tensorflow/... -//tensorflow/examples/android/..."` in this change so that `./configure` will work both on Linux and Windows. Do you think it's a reasonable change?

BTW, I am trying to resolve this failure:
http://ci.bazel.io/job/TensorFlow/BAZEL_VERSION=HEAD,PLATFORM_NAME=windows-x86_64/606/console